README.md

```diff
@@ -1,4 +1,22 @@
 # commitlint-config-conventional
 
-This is an extension of @commitlint/config-conventional for the purposes of
-overriding certain values as deemed necessary.
+This is an extension of the rules within @commitlint/config-conventional for
+the purposes of overriding certain values as deemed necessary. Those rules
+that this module is overriding are noted here:
+
+- `body-max-line-length` Increased to 300 characters to allow for commit messages that would otherwise exceed the 100 character limit.
+
+## Installation
+
+<pre>
```

Comment:
Also, worth mentioning this https://github.com/wagoid/commitlint-github-action#about-extends-in-your-config-file

As we use this action in many of our repos

Reply:
Thanks @tagoro9 . @shakefu suggested installing @commitlint/config-conventional directly, rather than via package.json, which works fine in some cases.